app/js/templates/categoryItem.hbs

```diff
@@ -0,0 +1,8 @@
+{{#each data}}
+    <div class="categoryItem">
+        <p>{{name}}   </p>
```

Comment:
why the big space after the name?

Reply:
To ensure space between category name and delete button without css - would you prefer us to do it with css?

Reply:
yes